app/assets/stylesheets/themes/scinote.scss

```diff
@@ -1727,6 +1727,17 @@ th.custom-field .modal-tooltiptext {
   max-height: 200px;
 }
 
+a.disabled {
```

Comment:
I just hope this doesn't affect buttons (which are often `<a>`-s) throughout the rest of the application.

Reply:
it not this what we want for a disabled element?